plugin/trino-bigquery/README.md

```diff
@@ -0,0 +1,29 @@
+# BigQuery Connector Developer Notes
```

Comment:
Can you please also mention how to reproduce the CI failure (how to reuse CI environment)?

Reply:
Can you explain a bit more about this? Do you mean how to create an environment which matches a failing test?